src/exporter.js

```diff
@@ -133,9 +136,22 @@ export default class {
   _tagsToStr(tags=[]){
     return tags.map(tag => tag.replace(/ /g, '_')).join(' ');
   }
-}
 
-export const rand = () => Math.random() * 100000000 | 0;
+  _getId(table, col, ts) {
+    const query = `SELECT ${col} from ${table} WHERE ${col} >= ${ts}`;
```

Comment:
>  (The greatest value + 1) === non-existing ID.
yep. But then we'll lost logic about datetime. Using max value was a continue of my question about non-random ids ( as result of using autoincrement ). 
Currently I'll reimplement original logic by link  you provide in issues discussion https://github.com/dae/anki/blob/24b451b0e4cfb50191e294052363a79f69f35c02/anki/utils.py#L193-L200